### ISSUE TITLE
helm s3 integration

### ISSUE THREAD

**Issue by sstarcher** [databus23/helm-diff#40]
Helm diff should play nicely with the helm s3 plugin. - https://github.com/hypnoglow/helm-s3

`helm fetch repo/chart --version 0.1.1` works cleanly, but 
`helm diff release repo/chart --version 0.1.1` fails

```
Command "helm diff" is deprecated, use "helm diff upgrade" instead
Error: Get s3://syapse-charts/charts/logstash-0.1.1.tgz: unsupported protocol scheme "s3"
Usage:
  diff [flags]
  diff [command]

Available Commands:
  upgrade     Show a diff explaining what a helm upgrade would change.
  version     Show version of the helm diff plugin

Flags:
  -h, --help                   help for diff
      --no-color               remove colors from the output
      --reset-values           reset the values to the ones built into the chart and merge in any new values
      --reuse-values           reuse the last release's values and merge in any new values
      --set stringArray        set values on the command line (can specify multiple or separate values with commas: key1=val1,key2=val2)
      --suppress stringArray   allows suppression of the values listed in the diff output
  -q, --suppress-secrets       suppress secrets in the output
  -f, --values valueFiles      specify values in a YAML file (can specify multiple) (default [])
      --version string         specify the exact chart version to use. If this is not specified, the latest version is used

Additional help topics:
  diff

Use "diff [command] --help" for more information about a command.

Error: plugin "diff" exited with error
```

**Comment by databus23** [comment]
Hmm not sure why. `helm-diff` uses the same routines that helm uses internally: https://github.com/databus23/helm-diff/blob/master/cmd/helm.go#L85-L90. Superficially it looks like it should work.

**Comment by sstarcher** [comment]
I have not dug into the helm code, but maybe going through the downloader does not effect the plugins.

**Comment by databus23** [comment]
Can you verify this has been fixed in version 2.9.0+1 of this plugin?

**Comment by sstarcher** [comment]
@databus23 works great thanks
